nixos/modules/system/boot/grow-partition.nix

```
@@ -9,30 +9,49 @@
   ...
 }:
 
-with lib;
+let
+  cfg = config.boot.growPartition;
+  rootIsBtrfs = config.fileSystems.${cfg.mountPoint}.fsType or null == "btrfs";
+in
 
 {
   imports = [
-    (mkRenamedOptionModule [ "virtualisation" "growPartition" ] [ "boot" "growPartition" ])
+    (lib.mkRenamedOptionModule [ "virtualisation" "growPartition" ] [ "boot" "growPartition" "enable" ])
```

```suggestion
    (lib.mkRenamedOptionModule [ "virtualisation" "growPartition" ] [ "boot" "growPartition" "enable" ])
    (lib.mkRenamedOptionModule [ "boot" "growPartition" ] [ "boot" "growPartition" "enable" ])
```
I think this would help users more.